### DISCUSSION TITLE
Bootstrap Gamma Distribution

### DISCUSSION THREAD

**Discussion by AmaanAliK** [casact/chainladder-python#454] [Q&A] [answered]
Hey guys! I wanted to ask we can use a different distribution for the Bootstrap function, that being the gamma dist. instead of the ODP. Im trying to stochastically model Paid triangles and I think Gamma will be a better choice! Please let me know your thoughts and recommendations, thank you as always

**Comment by jbogaardt** [comment, marked as answer]
This is supported by Theory (refer to 3.4. The GLM Bootstrap Model in [Monograph 4](https://www.casact.org/sites/default/files/2021-02/04-shapland.pdf). However, the implementation here is limited to ODP because the algorithm exploits the relationship between the ODP and the basic chainladder to get at a simple/fast implementation. A more comprehensive framework would be needed to generalize beyond ODP. I think any implementation should be treated as its own separate estimator.

**Reply by AmaanAliK** [reply]
I see! So a separate estimator must be constructed for the Gamma Bootstrap then right? Im going to try my luck with it to see if its doable
